Stratis.Bitcoin.Features.BlockStore/BlockRepository.cs

```diff
@@ -71,40 +67,39 @@ public virtual BlockStoreRepositoryPerformanceCounter PerformanceCounterFactory(
             return new BlockStoreRepositoryPerformanceCounter();
         }
 
-        public virtual Task Initialize()
+        public virtual Task InitializeAsync()
         {
             this.logger.LogTrace("()");
-            var genesis = this.network.GetGenesis();
+            Block genesis = this.network.GetGenesis();
 
-            var sync = this.session.Execute(() =>
+            Task task = Task.Run(() =>
             {
-                this.session.Transaction.SynchronizeTables(this.tableNames.ToList());
```

Comment:
no need for `SynchronizeTables`?

Reply:
before there was one transaction,
ST is per transaction
so now you do ST on per transaction basis and you have to do it
just not like before, which was global